queries.sql

```diff
@@ -138,3 +138,16 @@ INNER JOIN specializations ON vets.id = specializations.vet_id WHERE animals.spe
 /* What specialty should Maisy Smith consider getting? Look for the species she gets the most */
 SELECT species.name AS species_name, COUNT(animals.id) AS number_of_animals FROM animals INNER JOIN species ON animals.species_id = species.id
 WHERE animals.owner_id = (SELECT id FROM owners WHERE full_name = 'Maisy Smith') GROUP BY species.name ORDER BY species_name DESC LIMIT 1;
+
+/* Day 5 project */
+SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYZE SELECT * FROM visits where animal_id = 4;
+SELECT * FROM visits where vet_id = 2;
+EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
+SELECT * FROM owners where email = 'owner_18327@mail.com';
+EXPLAIN ANALYZE SELECT * FROM owners WHERE email = 'owner_18327@mail.com';
+
+/* create index to decrease the execution time of the query */
+CREATE INDEX idx_visits_animal_id ON visits(animal_id);
+CREATE INDEX idx_visits_vet_id ON visits(vet_id);
+CREATE INDEX idx_owners_email ON owners(email);
```

Comment:
- [ ] You have all done an excellent job so far, and it's evident that you had some fun with the explain analyze feature. I'm confident that in the future, you will become adept at initiating investigations for slow DB queries. However, to ensure efficient data access from the start, please remember to include the `CREATE INDEX` statements in the `schema.sql` file, as they define the database schema and optimize query performance.
- Here's the project requirement as a reminder:
  ![image](https://github.com/sadaf-Daneshgar/vet_clinic_database/assets/104968236/84d30b13-f068-4983-965d-d50d87dcf543)

Reply:
Hello @AbelG101  thanks for your nice review I did not get what should I do, should I just add this code into schema.sql file, it is enough?

Reply:
Hello @sadaf-Daneshgar, yes please place it in the `schema.sql` file instead of the `queries.sql`.

Reply:
ok thanks sir 👍